samples/cpp/benchmark_app/result_dump.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "result_dump.hpp"
+
+#include <nlohmann/json.hpp>
+
+#include "samples/slog.hpp"
+
+void ResultDump::add_result(const std::string& input_image, const ov::TensorVector& outputs) {
```

Comment:
compare_and_save_result() will be more readable?

Reply:
ok, I will modify it.